frontend/ui/components/Topbar/SelectNearNetwork/SelectNearNetwork.jsx

```diff
@@ -1,14 +1,21 @@
-import { useMemo } from 'react';
 import { useMatch, useNavigate, useParams } from 'react-router-dom';
-import { useStoreEffect } from '../../../../../react-vault/index.js';
+import { useStoreEffect, useStoreState } from '../../../../../react-vault/index.js';
 import { useLoader } from '../../../hooks/useLoader.js';
-import { useGetOptions } from './useGetOptions.jsx';
-import cn from './SelectNearNetwork.module.scss';
+import { Selector } from '../_general/Selector/Selector.jsx';
+
+const getOptions = (ids) =>
+  ids.map((networkId) => ({
+    label: networkId,
+    value: networkId,
+    networkId: networkId,
+  }));
 
 export const SelectNearNetwork = () => {
   const { spaceId, networkId } = useParams();
   const getAll = useStoreEffect((store) => store.nearProtocol.networks.getAll);
-  const options = useGetOptions(spaceId, networkId);
+  const ids = useStoreState((store) => store.nearProtocol.networks.ids);
```

Comment:
Why you don't use useGetOptions hook and instead doing all logic inside the component? It's much better to separate get options/defaul value logic into standalone function